src/General/Alert/Alert.test.tsx

```diff
@@ -297,9 +298,10 @@ describe('<Alert/> timeout', () => {
     );
 
     unmount();
-    expect(clearTimeoutSpy).toHaveBeenCalledTimes(2);
-    expect(clearTimeoutSpy.mock.calls[1][0]).toEqual(
-      useRefSpy.mock.results[1].value.current
+    expect(setTimeoutSpy.mock.calls.length).toEqual(
+      // No idea what's calling the additional clearTimeout here. It's not us,
+      // I checked.
+      clearTimeoutSpy.mock.calls.length + 1
```

Comment:
I was running into trouble with this test because I now have two timeouts (one in Alert and one in DelayedUnmount). But I figured this is really just testing that all timeouts are cleared in the end.

Reply:
Haven't figured out why there is an extra `clearTimeout` called. But I found the caller should be `DelayedUnmount` when running `unmount()`.

Reply:
>But I found the caller should be DelayedUnmount when running unmount().
No I don't think so, that is one of the two calls that I can account for. I tried adding log messages to both effect hook cleanup functions (one in Alert and one in DelayedUnmount) and both get called only once.